platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/CryptoBench.java

```diff
@@ -34,7 +34,7 @@
 @State(Scope.Thread)
 @Warmup(iterations = 0)
 @Measurement(iterations = 1)
-public abstract class CryptoBench extends VirtualMapBench {
+public class CryptoBench extends VirtualMapBench {
```

Comment:
Interesting. This class used to be really abstract (had at least one abstract method), but it is no longer. I'd probably leave it as is in this PR, and filed a separate small ticket to refactor CryptoBench / CryptoBenchMerkleDb